src/index.html

```diff
@@ -15,8 +15,25 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="http://127.0.0.1:5500/src/index.html"><img class="img" src="images/logo.png" alt="logo"></a>
```

Comment:
Consider using a relative URL for the logo image source to ensure it works correctly regardless of the server or domain where the HTML file is hosted. This will make the path more flexible and adaptable to different environments.